### ISSUE TITLE
Would it be possible to read/write vertex data 

### ISSUE THREAD

**Issue by SteffenPL** [GenevieveBuckley/napari-meshio#3]
Thanks for the great plugin!

I wanted to ask if it is possible to extend meshio to read/write [vertex data](https://napari.org/stable/howtos/layers/surface.html#surface-data) (e.g. color data or labels)? 

I guess only some formats might support this... (I could also try to make a PR, but I don't know yet how to implement it.)

**Comment by GenevieveBuckley** [comment]
Yes, it could be possible. The meshio `Mesh` class has both `mesh.point_data` and `mesh.cell_data` attributes, that might be what you want. See [here in the meshio README](https://github.com/nschloe/meshio/blob/0138cc8692b806b44b32d344f7961e8370121ff7/README.md?plain=1#L124-L127).

The difficult part is that `mesh.point_data` and `mesh.cell_data` are dictionaries, so it's quite flexible with the formatting of what could be stored in there. It might be difficult to check whether it's the right type of information you want to pass to napari.

For the napari-meshio reader function, you would likely need to pass the extra information to napari with [this line](https://github.com/GenevieveBuckley/napari-meshio/blob/f4a0d6b079adc1119f4b920f60aa64fffde9ed37/src/napari_meshio/_reader.py#L72). You will probably need to convert the information from the meshio `mesh.point_data` dictionary into something that you can give to the [vertex_values](https://napari.org/stable/api/napari.layers.Surface.html#napari.layers.Surface.vertex_values) in napari surfaces.

I don't think any change will be needed for the writer part of this plugin. Hopefully that should already work.

Finally, it is possible that not all the [underlying file formats](https://github.com/GenevieveBuckley/napari-meshio/blob/f4a0d6b079adc1119f4b920f60aa64fffde9ed37/src/napari_meshio/_available_file_formats.py#L7-L47) support this. If they don't support it, that information will be lost. However, the .vtk file format definitely *does* support this (I'm not sure about the others).
